test/integration-test.ts

```diff
@@ -14,12 +14,16 @@ const wallet = Wallet.generateWalletFromSeed('snYP7oArxKepd3GPDcrjMsJYiJeJB')!
 
 // A hash of a successfully validated transaction.
 const transactionHash =
-  '9A88C8548E03958FD97AF44AE5A8668896D195A70CF3FF3CB8E57096AA717135'
+  '95D30FED388C1B4BC1CC73BE3B8D79875995A26F7FB46061CF1A77972AEB9A7A'
 
 // A XpringClient that makes requests. Uses the legacy protocol buffer implementation.
 const legacyGRPCURL = 'grpc.xpring.tech:80'
 const legacyXpringClient = new XpringClient(legacyGRPCURL)
 
+// A XpringClient that makes requests. Uses rippled's gRPC implementation.
+const rippledURL = '3.14.64.116:50051'
```

Comment:
Is it ok to hardcode this IP?

Reply:
This is a server that I am running and admin-ing outside of the ripple network as we are unsure if we want to open gRPC ports on a TestNet node.